packages/x-components/src/x-modules/related-prompts/components/related-prompt.vue

```diff
@@ -1,38 +1,26 @@
 <template>
-  <div
-    class="x-related-prompt x-bg-neutral-10 x-p-24 x-flex x-flex-col x-gap-16"
-    data-test="related-prompt"
-  >
-    <div class="x-related-prompt__info x-flex x-flex-col x-gap-16">
+  <div class="x-related-prompt" data-test="related-prompt">
+    <div class="x-related-prompt__info x-flex x-flex-col">
       <slot name="header" :suggestionText="relatedPrompt.suggestionText">
         {{ relatedPrompt.suggestionText }}
       </slot>
       <slot name="next-queries" :nextQueries="relatedPrompt.nextQueries">
         <SlidingPanel :resetOnContentChange="false">
-          <div class="x-flex x-gap-8 x-pr-8">
+          <div class="x-flex x-gap-8">
             <button
               v-for="(nextQuery, index) in relatedPrompt.nextQueries"
               :key="index"
               @click="onClick(nextQuery)"
-              class="x-button x-button-lead x-button-sm x-button-outlined x-rounded-sm x-border-lead-50 x-text-neutral-75 hover:x-text-neutral-0 selected:x-text-neutral-0 selected:hover:x-bg-lead-50"
-              :class="{ 'x-selected': selectedNextQuery === nextQuery }"
+              :class="[{ 'x-selected': selectedNextQuery === nextQuery }, nextQueryButtonClass]"
```

Comment:
Hi, I would name it `buttonClass` so it is more agnostic about its content (who knows if they will finally be next queries or another kind of suggestion in the future)

Reply:
There are more than one button, I think it's better to have a more clear name for this